SendSignatureRequestMessage/function.json

```diff
@@ -4,7 +4,7 @@
       "name": "sendSignatureRequestMessage",
       "type": "queueTrigger",
       "direction": "in",
-      "queueName": "waiting-for-signature",
+      "queueName": "%WaitingMessageQueueName%",
```

Comment:
I think it's %QUEUE_STORAGE_NAME_FOR_WAITING_MESSAGE% (local settings isnt' read in production)

Reply:
`QUEUE_STORAGE_NAME_FOR_WAITING_MESSAGE` is only a placeholder for local settings. The real env name is `WaitingMessageQueueName`. https://github.com/pagopa/io-functions-sign/blob/187b304741b898e0589b54dfc3fb777b41c64051/src/infra/azure/storage/config.ts#L30